### ISSUE TITLE
Improve the Main Landing page of 'Mutual Aid World' app.

### ISSUE THREAD

**Issue by Erioldoesdesign** [factn/Resilience-Design#43]
This is an epic which collects the issues which are around improving the main 'landing page' once a user hits the Mutual Aid World web app.

In both mobile views, tablet and desktop.

https://mutualaid-757f6.web.app/

![Screenshot 2021-02-09 at 21.13.17.png](https://images.zenhubusercontent.com/5a6a0d5a4b5806bc2bc44bb9/83f91899-64ba-4b34-8ced-863c2867f86f)

**Comment by joycekimdesigns** [comment]
I did some competitive research to see what other similar websites had on their website to see what we are missing. I started mapping out what the page would be for desktop and tablet views, but wanted to focus on mobile views first.
![Competitive Research:Sketches](https://user-images.githubusercontent.com/78825537/108586357-b6dd0900-7302-11eb-9ee0-71c7e8ea5e35.jpg)

While doing an audit on the the current landing page, I noticed four major points that I thought should be addressed along with information we could possibly add.
![Landing Page Audit](https://user-images.githubusercontent.com/78825537/108586479-b133f300-7303-11eb-8073-c6234d90c2e5.jpg)
![Competitive Research](https://user-images.githubusercontent.com/78825537/108586414-41256d00-7303-11eb-8d77-2b967b8818fc.jpg)

Wireframe options:
**Option 1**
![Uploading Wireframe 1.jpg…]()

**Option 2**
![Wireframe 2](https://user-images.githubusercontent.com/78825537/108586508-df193780-7303-11eb-93eb-3af4b0c6b39e.jpg)

**Option 3**
![Wireframe 3](https://user-images.githubusercontent.com/78825537/108586514-e80a0900-7303-11eb-85c4-4b9b2c3bcacd.jpg)

**Option 4**
![Wireframe 4 1](https://user-images.githubusercontent.com/78825537/108586522-f1937100-7303-11eb-9388-079c71efe0b7.jpg)
Expanded View:
![Wireframe 4 2](https://user-images.githubusercontent.com/78825537/108586532-fc4e0600-7303-11eb-9ecd-4f48840fab03.jpg)

Figma File: [https://www.figma.com/file/EmaLouXw7xH1qp7LjTAww5/Mutual-Aid-Feed-Folks-Main-Landing-Page?node-id=0%3A1](url)

**Comment by thoughtsky** [comment]
> I did some competitive research to see what other similar websites had on their website to see what we are missing. I started mapping out what the page would be for desktop and tablet views, but wanted to focus on mobile views first.
> 
> 
> While doing an audit on the the current landing page, I noticed four major points that I thought should be addressed along with information we could possibly add.
> 
> 
> 
> Wireframe options:
> Option 1
> 
> 
> Option 2
> 
> 
> Option 3
> 
> 
> Option 4
> 
> Expanded View:
> 
> 
> Figma File: https://www.figma.com/file/EmaLouXw7xH1qp7LjTAww5/Mutual-Aid-Feed-Folks-Main-Landing-Page?node-id=0%3A1

Really good job there! I personally really like Option 2. I think it provides the necessary clearness and is straight to the point!

**Comment by Erioldoesdesign** [comment]
Hello @joycekimdesigns there's some definite improvement here regarding the general UX layout for sure. I think critically, having a key branding colour that is well contrasted with white/light greys will help with accessibility as well as generally, how easily users read the content.

If it's okay with you, I'll drop some comments directly in your Figma file but also summarise them here in another comment too.

Just to note your research is excellent! very thorough! 😄

**Comment by joycekimdesigns** [comment]
Hi @Erioldoesdesign this link should work, sorry about that: [https://www.figma.com/file/EmaLouXw7xH1qp7LjTAww5/Mutual-Aid-Feed-Folks-Main-Landing-Page?node-id=0%3A1](url)

I'll wait for your feedback before I start working on it again, thanks! 😃

**Comment by joycekimdesigns** [comment]
Based on the comments I received, I tried adding some color to the design to have a better feel:
![Wireframe 6](https://user-images.githubusercontent.com/78825537/111060619-18504d80-8453-11eb-9513-5e448ed31365.jpg)

It stood out to me that dividing up the sections with a box made the page feel really choppy, and didn't feel very welcoming despite the colors. I decided to explore the curves in the logo and tie that into the design of the page:
A:
![21 - A](https://user-images.githubusercontent.com/78825537/111060707-ea1f3d80-8453-11eb-9241-1c67f5e8ff6e.jpg)

B:
![21 - B](https://user-images.githubusercontent.com/78825537/111060710-eb506a80-8453-11eb-941c-a25411888335.jpg)

C:
![21 - C](https://user-images.githubusercontent.com/78825537/111060712-ef7c8800-8453-11eb-87c7-45a1b3389cf4.jpg)

D:
![21 - D](https://user-images.githubusercontent.com/78825537/111060718-f2777880-8453-11eb-90a7-14e827549d6a.jpg)

E:
![21 - E](https://user-images.githubusercontent.com/78825537/111060723-f6a39600-8453-11eb-90fb-977b8e94411a.jpg)

F:
![21 - F](https://user-images.githubusercontent.com/78825537/111061371-e4c3f200-8457-11eb-9ba8-98aaf9223c61.jpg)

**Comment by Erioldoesdesign** [comment]
Just updating a comment here saying myself and Joyce spoke about:

1. How this could be developed with [CSS shapes](https://www.w3schools.com/howto/howto_css_shapes.asp) that might make it lighter file size wise and 'responsive'
2. We Also spoke about how it could be done through SVG.
3. I suggested Joyce work on a 'simpler' shapes version with more standard 'circles and triangles and rectangles lines' as an 'easier to dev' option.
4. We spoke about a 'footer' issue here: https://github.com/factn/Resilience-Design/issues/47
5. We spoke about the next work to focus on.

**Comment by joycekimdesigns** [comment]
3/26/21 Meeting Recap
Participants: Eriol & Joyce

- Discuss what developing design with CSS shapes mean
- Increase size of smallest dimensions (minimum 40px)
- Fonts can change (some options: Lato, Fira, Open Sans...) 
- Drop shadows can be done in CSS
- Next sizes to explore: Tablet (portrait and landscape)
- Refer to https://www.figma.com/file/GLg0zPNnwMEGIb9TytCrxQ/Resilience-App?node-id=37375%3A286 
     - MVP v1: things that didn't get implemented
     - MVP v0: use things from here
- Discuss Footer Info, see Issue #47

**Comment by joycekimdesigns** [comment]
Based on feedback and in an effort to simplify the shapes, I have worked on a few more designs. 
FYI, font and tablet size have not been explored yet. Those changes in addition to using information from MVP v0 will be added once a direction is set.

Design D.1:
![21 - Option D 1 Background Only  Simplified](https://user-images.githubusercontent.com/78825537/113271234-21cd1880-928f-11eb-9e85-c43d1684fc11.jpg)

**Comment by joycekimdesigns** [comment]
Design B: 
![21 - Option D 3 Background Only  Simplified](https://user-images.githubusercontent.com/78825537/113271030-f5190100-928e-11eb-9389-8cbf401f9894.jpg)

**Comment by joycekimdesigns** [comment]
Design C: 
![21 - Option D 2 Background Only  Simplified](https://user-images.githubusercontent.com/78825537/113271087-ffd39600-928e-11eb-807c-7d34c9ccaa9c.jpg)

**Comment by joycekimdesigns** [comment]
Design D.2:
![21 - Option D 2 Background Only  Simplified](https://user-images.githubusercontent.com/78825537/113271342-46c18b80-928f-11eb-9d9c-f42a8fd45a92.jpg)

**Comment by joycekimdesigns** [comment]
Design D.3:
![21 - Option D 3 Background Only  Simplified](https://user-images.githubusercontent.com/78825537/113271392-550fa780-928f-11eb-909d-c0680fed42bc.jpg)

**Comment by joycekimdesigns** [comment]
Design D.4:
![21 - Option D 4 Background Only  Simplified](https://user-images.githubusercontent.com/78825537/113271420-5fca3c80-928f-11eb-8b2b-e96e9997dbca.jpg)

**Comment by thoughtsky** [comment]
Hey! Absolutely great job. Personally I think D.2 cuts it. It has a clear separation of sections and the visual language is clear. Only thing that confused me was the "Donate" section having a button called "Volunteer". I know there is some discussion around the terms but wouldn't volunteer involve helping out "physically" while donating would be another way of helping out?

**Comment by Erioldoesdesign** [comment]
Agreed! D2 is the one I like with the circle at the bottom for the call us info.

So I think the next part of this work is to find the rote through to development. @utunga any ideas?

**Comment by joycekimdesigns** [comment]
5/5/21 -  Hi All, I've added iphone, ipad, and desktop views for the landing page.
iPhone: 
![050521_MutualAid_iPhone](https://user-images.githubusercontent.com/78825537/117218779-ab08cb00-adb8-11eb-826d-dd1bad6b790d.png)

**Comment by joycekimdesigns** [comment]
iPad:
![050521_MutualAid_iPad](https://user-images.githubusercontent.com/78825537/117218851-d5f31f00-adb8-11eb-97f8-5d2ca5b82084.png)

**Comment by joycekimdesigns** [comment]
Desktop:
![050521_MutualAid_Desktop](https://user-images.githubusercontent.com/78825537/117218869-e1dee100-adb8-11eb-8ae7-9876681a6340.png)

**Comment by Erioldoesdesign** [comment]
LOVE THESE!
